### ISSUE TITLE
Google analytics results in REST services error

### ISSUE THREAD

**Issue by bbraspenning** [magento/magento2#3180]
Google analytics results in REST services error

When having google analytics configured
    configuration/sales/google analyitics
        enabled yes
        account nr  99999
        enable content experiments yes

using the following url rest/V1/categories
with the following message
    {  
   "category":{  
      "path":"1",
      "parent_id":2,
      "name":"DISCOVERY",
      "is_active":"1",
      "include_in_menu":"1",
      "custom_attributes":[  
         {  
            "attribute_code":"url_key",
            "value":"103-DISCOVERY"
         },
         {  
            "attribute_code":"is_anchor",
            "value":"1"
         }
      ]
   }
    }

Gives the following error message:  
    {
    "message": "Could not save category: %1",
    "parameters": ["Wrong request parameters"]
    }

```
==> origin: abstract save 
protected function _initRequestParams()
{
    $params = $this->_request->getParam('google_experiment');
    if (!is_array($params) || !isset($params['experiment_script']) || !isset($params['code_id'])) {
        throw new \InvalidArgumentException('Wrong request parameters');
    }
    $this->_params = $params;
}
```

Same issue occurs when trying to update products through REST call or changing position through rest call. When disabling google analytics, the rest  calls work again. How come?

**Comment by rklerck** [comment]
I have the same problem!

**Comment by vrann** [comment]
@bbraspenning Thanks for reporting, seems like an issue in GoogleAnalytics or Categories service

**Comment by katmoon** [comment]
Hi @bbraspenning 

Internal ticket `MAGETWO-50414` has been logged for further investigation.

Thanks,
Kateryna.

**Comment by mslabko** [comment]
Hello @bbraspenning 
Issue was fixed on develop branch (https://github.com/magento/magento2/commit/d58c63eb5915d6c460ccf3526cce1f8828d3f4ae) and will be available in 2.1 version
Thank you for reporting!
